addons/mail/static/tests/message/message.test.js

```diff
@@ -1993,3 +1994,23 @@ test("Copy Message Link", async () => {
     await press("Enter");
     await contains(".o-mail-Message", { text: url(`/mail/message/${messageId_2}`) });
 });
+
+test("display the notification message's posting date", async () => {
+    mockDate("2025-01-01 12:00:00", +1);
+    const pyEnv = await startServer();
+    const channelId = pyEnv["discuss.channel"].create({ name: "Hogwarts" });
+    const partnerId = pyEnv["res.partner"].create({ name: "Tom Riddle" });
+    const userId = pyEnv["res.users"].create({
+        name: "Harry Potter",
+        partner_id: partnerId,
+    });
+    await start();
+    await openDiscuss(channelId);
+    await withUser(userId, () => {
+        getService("orm").call("discuss.channel", "add_members", [[channelId]], {
+            partner_ids: [partnerId],
+        });
+    });
+    await contains(".o_mail_notification", { text: "Tom Riddle joined the channel" });
+    await contains(".o-mail-NotificationMessage-date", { text: "Today at 1:00 PM" });
```

Comment:
I would make a single assertion `.o-mail-NotificationMessage:contains('Tom Riddle joined the channel\nToday at 1:00 PM');`